k8s/production/runners/protected/graviton/3/release.yaml

```diff
@@ -48,7 +48,7 @@ spec:
           [runners.kubernetes]
             helper_image = "gitlab/gitlab-runner-helper:arm-latest"
             privileged = false
-            helper_memory_request = "512m"
+            helper_memory_request = 512
```

Comment:
Why this change? Does the "512m" string not work?

Reply:
The rest of the runners in `k8s/production/runners/*` have this as a string, so I think before merging we should either back out this change, or update the rest of the runner memory requests to also be an int. Maybe the former is better, as the gitlab docs usually have these types of fields set as strings? (see `helper_cpu_limit` in this example - https://gitlab.com/gitlab-org/gitlab-runner/blob/ffdb32cb9cd1e325b5018cea2ddd6084b6ec741e/docs/executors/kubernetes.md#define-keywords-in-the-config-toml)